test/verify/check-client

```diff
@@ -241,6 +241,17 @@ Command = /usr/bin/env python3 -m cockpit.beiboot
         b.set_val("#conversation-input", "wrong")
         b.click("#login-button")
         b.wait_val("#conversation-input", "")
+
+        # quirk: this test really runs the "bastion host" code path in beiboot, as we are not actually
```

Comment:
I'd prefer if we hack the filesystem from the test case to make this run the flatpak code path instead...